src/components/Marketplace/FarmCard.js

```diff
@@ -0,0 +1,228 @@
+import React, { useState } from 'react';
+import './FarmCard.css';
+import {
+  Card, CardContent, Chip, Collapse, Grid, IconButton,
+} from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import ExpandLessIcon from '@material-ui/icons/ExpandLess';
+import CheckCircleOutlineIcon from '@material-ui/icons/CheckCircleOutline';
+import PropTypes from 'prop-types';
+
+/* Custom styling for material-ui components */
+const useStyles = makeStyles({
+  cardContainer: { // Overall card container
+    background: 'white', // change to lightblue for testing
```

Comment:
Thanks for this note! :) Make sure to try to use the exact hex color found in Figma